src/frontend/app.ts

```diff
@@ -133,19 +133,22 @@ export default (octokitApp: App): Express => {
 		syncStatus === "COMPLETE" ? "Connected" : "Connect"
 	);
 
-	app.use("/public", express.static(path.join(rootPath, "static")));
+	app.use("/public", elapsedTimeMetrics("/public/*"), express.static(path.join(rootPath, "static")));
 	app.use(
 		"/public/css-reset",
+		elapsedTimeMetrics("/public/*"),
```

Comment:
There's no point of having these individual ones as the `use` above (L136) will hit all of them.

Reply:
Oh interesting, I thought this line was overriding it. 

How does it work? Would it only override the last handler for this route( `express.static(path.join(rootPath, "static")`  will be replaced with `express.static(
			path.join(rootPath, "node_modules/@atlaskit/css-reset/dist")
		) ` ?